src/Templates/translatable/widgets/changeDetails.twig

```diff
@@ -394,15 +428,6 @@
             </option>
         </select>
     </div>
```

Comment:
posunutý ten file upload je tu schválně?

Reply:
jo, aby to bylo vidět, že tam přibyl, když zadáš to datum narození, je to pak více "interaktivní".
Nemusí to tam být, jen se mi ta stránka potom takhle líbila víc.